instances/etcd/cloud_init/bootstrap.template.sh

```diff
@@ -62,7 +69,3 @@ done
 
 # put the flannel config in etcd
 curl -sf -L http://$FQDN_HOSTNAME:2379/v2/keys/flannel/network/config -X PUT --data-urlencode value@/tmp/flannel-network.json
```

Comment:
Did you verify that the home dir permissions are correct before removing this?  ls -la /home/  to make sure opc isn't owned by root

Reply:
Yes I did. I think Ubuntu had some sort of bug.

Reply:
sounds good.